packages/nc-gui/components/smartsheet/header/CellIcon.ts

```diff
@@ -81,6 +81,8 @@ const renderIcon = (column: ColumnType, abstractType: any) => {
     return iconMap.percent
   } else if (isInt(column, abstractType) || isFloat(column, abstractType)) {
     return iconMap.number
+  } else if(column.uidt === UITypes.Geometry) {
```

Comment:
Define `isGeometry` in `packages/nc-gui/utils/cell.ts` and use it here instead. keep the pattern consistent.